starboard/rdk/launcher.py

```diff
@@ -172,6 +172,14 @@ def _request_payload(method):
           f'"configuration":{{ {cmd_test_param}}} }} }}\' ')
       return json_cmd
 
+    # test file preparation
+    rdk_test_symlink_dir = '/usr/share/content/data/app'
+    rdk_tmp = '/var/lib/persistent/rdkservices/Cobalt/Cobalt/.cobalt_storage'
+    self.test_prep_command = (f'rm -rf {rdk_tmp}; '
+                              f'rm -rf {rdk_test_symlink_dir}/cobalt; '
+                              f'ln -s {rdk_test_dir}/cobalt '
```

Comment:
I think this symlink deserves a comment explaining why it exists.

Reply:
added.